### DISCUSSION TITLE
Manual deploy on web breaks (not reading secrets), but CLI deploy work fine ?

### DISCUSSION THREAD

**Discussion by gcoch** [observablehq/framework#1863] [Q&A] [answered]
I'm running 1.13.0. When I deploy from the CLI, the secrets I use to access remote file (in Digital Ocean) works as expected. In my development environment I have exported the secrets.

If click Manual Deploy from the web interface the secrets that I have stored in the Secrets tab are not read, so the app breaks (i.e. it doesn't access the remote files). 

Any ideas/suggestions?

**Comment by mythmon** [comment, marked as answer]
Hi Gabriel, lets see if we can figure out what's going on. Can you share the part of your data loader that is accessing the secrets?

Also, it would be helpful to have a screenshot of the UI where you set up the secrets. If you prefer, you can send that to my email, michael@observablehq.com

**Reply by gcoch** [reply]
Thanks @mythmon. This issue was resolved by deleting the keys and adding them again.
